avocado-setup.py

```diff
@@ -159,6 +159,11 @@ def env_check(disable_kvm):
                 "No dependancy check flag is set, proceeding with bootstrap")
             logger.info("Please install following "
                         "dependancy packages %s", " ".join(not_found))
+        elif args.install_deps:
+            logger.warning("Installing packages %s", " ".join(not_found))
+            if helper.install_packages(not_found):
+                logger.error("Some packages not installed")
```

Comment:
can we print which packages not installed , that help user to identify what was wrong

Reply:
@PraveenPenguin that is being printed from the `helper.install_packages()`

Reply:
ah ok sure i over looked that code :(